samples/dotnet/KernelBuilder/Program.cs

```diff
@@ -4,6 +4,7 @@
 // The easier way to instantiate the Semantic Kernel is to use KernelBuilder.
 // You can access the builder using either Kernel.Builder or KernelBuilder.
 
+#pragma warning disable CA1852
```

Comment:
Disabled due to a bug in CA1852 for the latest .net 7 release.

https://github.com/dotnet/roslyn-analyzers/issues/6141